Basic-Scripts/anagram1.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 25 11:31:58 2018
+
+@author: Gulshan
+"""
+
+
+str1 = input("Enter the first string: ")
+str2 = input("Enter the second string: ")
+
+count1 = 0
+i = 0

```

Comment:
@gulshanbaraik01 Change the alphabets to meaningful variable names

Reply:
> @gulshanbaraik01 It would be great if you could raise one file per PR.
> If you have 3 different approaches for a similar problem, implement the only one which u think is better time & space complexity wise.

Implement this thing @gulshanbaraik01 
Can't merge 3 different files.